salt/monitoring/prometheus/rules.yml

```
@@ -1,28 +1,83 @@
 groups:
-- name: primary-resource-saphana-down
+- name: sap-hana-resource-monitoring
   rules:
-  - alert: SAPHANAMasterResourcesDown
-    expr: absent(cluster_node_resources{resource_name="rsc_saphana_prd_hdb00",role="master"})
+  - alert: sap-hana-master-resource-down
+    expr: absent(cluster_node_resources{resource_name=~"rsc_saphana_prd_hdb.*",role="master"})
```

@MalloZup the `prd` part is also dynamic. 

```suggestion
    expr: absent(cluster_node_resources{resource_name=~"rsc_saphana_.*",role="master"})
```